content/_releases/24/17.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Product Update - v24.17
+description: Details of the product update v24.17 from the 27th of April 2024.
+layout: rel
+version: "24.17"
+section: Product Updates
+category: product-updates
+releaseDate: 2024-04-27
+redirect_from:
+  - /releases/24.17.html
+---
+
+## New Features
+### New Quota for the number of requests to Webhooks per Contract/Workspace
+We have implemented new quota types to enhance control over webhook calls per minute:
+* `per_contract_request_webhooks_per_minute_limit`: controls calls to webhooks per minute at the contract level.
+* `per_workspace_request_webhooks_per_minute_limit`: controls calls to webhooks per minute at the workspace level.
+
+These quotas enable fine-grained management of webhook usage, ensuring optimal performance and resource utilization.
+
+**Error notification for Quota exceedance**
+
+When the quota is exceeded, the following error message will be displayed:
+```
+{
+  "error": "Request quota exceeded: contract 5b5ed1cf272cf80011ae7b43 (false) | workspace 59d341e9037f7200184a408b (true)",
+  "requestId": "9a315b2c476bd2c6f40a8d739d3f348e"
+}
+```
+Keep an eye on your quota limits to maintain smooth operations!
+
+### New feature-flag for error subscription management
+We've introduced a new feature-flag `subscribe_to_error_by_default`, designed to streamline error subscription management within flows. This feature is accessible at both Tenant and Contract levels.
+By default, this feature is set to `true` for all tenants, ensuring seamless error tracking. You can customize this setting using API calls through `POST` and `PATCH` methods at the following endpoints:
+* v2/tenants/:id
+* v2/contracts/:id
```

Comment:
add links https://api.elastic.io/docs/v2#/contracts and https://api.elastic.io/docs/v2#/tenants